packages/react-native-reanimated/src/layoutReanimation/animationBuilder/ComplexAnimationBuilder.ts

```diff
@@ -195,45 +194,61 @@ export class ComplexAnimationBuilder extends BaseAnimationBuilder {
   }
 
   /**
-   * Lets you adjust the rest displacement threshold of the spring animation.
-   * Can be chained alongside other [layout animation
-   * modifiers](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/glossary#layout-animation-modifier).
-   *
-   * @param restDisplacementThreshold - The displacement below which the spring
-   *   will snap to the designated position without further oscillations.
+   * @deprecated Use {@link energyThreshold} instead. This method currently does
+   *   nothing and will be removed in the upcoming major version.
    */
   static restDisplacementThreshold<T extends typeof ComplexAnimationBuilder>(
     this: T,
-    restDisplacementThreshold: number
+    _restDisplacementThreshold: number
   ) {
-    const instance = this.createInstance();
-    return instance.restDisplacementThreshold(restDisplacementThreshold);
+    return this.createInstance();
+  }
+
+  /**
+   * @deprecated Use {@link energyThreshold} instead. This method currently does
+   *   nothing and will be removed in the upcoming major version.
+   */
+  restDisplacementThreshold(_restDisplacementThreshold: number) {
+    return this;
+  }
+
+  /**
+   * @deprecated Use {@link energyThreshold} instead. This method currently does
+   *   nothing and will be removed in a future version.
+   */
+  static restSpeedThreshold<T extends typeof ComplexAnimationBuilder>(
+    this: T,
+    _restSpeedThreshold: number
+  ) {
+    return this.createInstance();
   }
 
-  restDisplacementThreshold(restDisplacementThreshold: number) {
-    this.restDisplacementThresholdV = restDisplacementThreshold;
+  /**
+   * @deprecated Use {@link energyThreshold} instead. This method currently does
+   *   nothing and will be removed in a future version.
+   */
+  restSpeedThreshold(_restSpeedThreshold: number): this {
     return this;
   }
 
   /**
-   * Lets you adjust the rest speed threshold of the spring animation. Can be
-   * chained alongside other [layout animation
+   * Lets you adjust the energy speed threshold level to stop the spring
```

Comment:
```suggestion
   * Lets you adjust the energy threshold level to stop the spring
```

`energy speed threshold`? Is this intended?

Reply:
This is a typo, thanks for pointing it out!